### PR TITLE
test framework done, adding asserts

### PR DESCRIPTION
**Pull request by richardechegaray**

### CODE REVIEW

File: frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            onView(withIndex(withId(R.id.card_view), 0)).perform(swipeLeft());
+            sleep(1000);
+            onView(withIndex(withId(R.id.card_view), 0)).perform(swipeRight());
+        }
+
+        sleep(1000);
+
+        onView(withId(R.id.first_place_idea_text)).check(matches(withText("Horror")));
+
+        ViewInteraction materialButton4 = onView(
```
Comment (paullin03) [inline frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java:183]: Change variable name here

File: frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+            @Override
+            public boolean matchesSafely(View view) {
+                return matcher.matches(view) && currentIndex++ == index;
+            }
+        };
+    }
+
+    @Rule
+    public ActivityScenarioRule<MainActivity> mActivityTestRule = new ActivityScenarioRule<>(MainActivity.class);
```
Comment (paullin03) [inline frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java:86]: Prefer rule + test at the top ahead on top of the helpers - newspaper style

File: frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java (hunk 1 above)
Comment (richardechegaray) [reply frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java:183]: resolved

File: frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java (hunk 2 above)
Comment (richardechegaray) [reply frontend/app/src/androidTest/java/com/quickpick/SwipeTest.java:86]: resolved

**Review by richardechegaray — COMMENTED** [review]
i cannot approve my own request

**Pull request merged**